### DISCUSSION TITLE
How would I make it so that player 1 controls one character and player 2 controls the other

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#161] [Q&A] [answered]
How would I make it so that player 1 controls one character and player 2 controls the other I was asking this question because I am making 2player and 4player split screen

**Comment by r6915ee** [comment]
> How would I make it so that player 1 controls one character and player 2 controls the other I was asking this question because I am making 2player and 4player split screen

I guess maybe perhaps making it so that each input in the player scripts can be react to a different version of each input? I'm not sure what you're trying to achieve with this.

**Reply by Raptorq1** [reply]
Iam trying to make local split screen for my fan game iam working on

**Comment by DimensionWarped** [comment, marked as answer]
> How would I make it so that player 1 controls one character and player 2 controls the other I was asking this question because I am making 2player and 4player split screen

This is... how things already work. Currently if we have two characters, player 1 controls Sonic and player 2 controls Tails. You shouldn't need to do anything to make that happen. If you modify your options to have different characters in slots 1 and 2, that maintains the same behavior... there really isn't anything needed here.
